docs/synthesis.md

```diff
@@ -0,0 +1,145 @@
+# Synthesis rules
+
+Synthesis is the process of creating entities from telemetry. Given some rules we try to match them against all the telemetry in order to create entities and create tags for them.
+
+Synthesis rules should be defined in the `definition.yaml` file under a `synthesis.rules` section.
+
+The requirement for a rule is to define the telemetry attribute that provides the `identifier` and the attribute that will be used as the `name` of the entity.
+
+```yaml
+synthesis:
+  rules:
+  - identifier: hostname
+    name: hostname
+```
+
+| **Name** | **Type** | **Description**  |
+| -------- | -------- | ---------------- |
+| name    | String | Required. The attribute to use for the entity name |
+| identifier| String| Required. Telemetry attribute to use as the entity identifier.|
+| encodeIdentifierInGUID | Boolean | Defaults to `false`. If true the identifier value will be encoded to respect the [GUID limits][guid_spec] |
+| conditions | List | Defaults to empty list. The list of conditions to apply in the data point to match the rule |
+| tags     | List   | Defaults to empty list. The list of attributes to copy as entity tags if the rule matches |
+
+### Conditions
+
+You can define extra conditions that must match in order to produce an entity.
+
+- Attribute must exist
+```yaml
+synthesis:
+  rules:
+  - identifier: hostname
+    name: hostname
+  	conditions:
+    - attribute: aws.az
+```
+
+- Attribute must have value
+```yaml
+synthesis:
+  rules:
+  - identifier: hostname
+    name: hostname
+  	conditions:
+    - attribute: aws.az
+      value: "us-west-1"
+```
+
+- Attribute must start with
+```yaml
+synthesis:
+  rules:
+  - identifier: hostname
+    name: hostname
+  	conditions:
+    - prefix: us-
+```
+
+
+| **Name** | **Type** | **Description**  |
+| -------- | -------- | ---------------- |
+| attribute | String  | Required. The name of the attribute to match in the data point. If only this field is defined the attribute must exist in the data point |
+| value    | String   | Optional. Can't be mixed with `prefix`. The exact value the attribute must contain in order to match the data point. |
+| prefix | String | Optional. Can't be mixed with `value`. The attribute must start with this value. |
+
+
+### Tags
+
+You can also define a set of attributes that can be copied into entity tags.
+These attributes will only be copied after a datapoint matches a rule.
+
+```yaml
+synthesis:
+  rules:
+  - identifier: hostname
+    name: hostname
+    tags:
+      aws.az:
+```
+
+If the attribute `aws.az` is present in the data point that matched the rule it's value will be copied into a tag named `aws.az` into the entity.
```

Comment:
```suggestion
If the attribute `aws.az` is present in the data point that matched the rule its value will be copied into a tag named `aws.az` into the entity.
```

Reply:
this has been rewrited by docs team so I'm "rejecting" the fix